plugins/kubernetes-react/src/components/Pods/PodDelete/PodDeleteButton.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { useState } from 'react';
+
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import CardActions from '@material-ui/core/CardActions';
+import Button from '@material-ui/core/Button';
+import DeleteIcon from '@material-ui/icons/Close';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import { useTranslationRef } from '@backstage/core-plugin-api/alpha';
+import { kubernetesReactTranslationRef } from '../../../translation';
+
+import { usePodDelete } from './usePodDelete';
+import { PodScope } from './types';
+
+/**
+ * Props for PodDeleteButton
+ *
+ * @public
+ */
+export interface PodDeleteButtonProps {
+  podScope: PodScope;
+  buttonText?: string;
+}
+
+/**
+ * a Delete button to delete a given pod
+ *
+ * @public
+ */
+export const PodDeleteButton = ({
+  podScope,
+  buttonText,
+}: PodDeleteButtonProps) => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [hasError, setHasError] = useState(false);
+  const deletePod = usePodDelete();
+
+  const { t } = useTranslationRef(kubernetesReactTranslationRef);
+  const getButtonText = buttonText ?? t('podDrawer.buttons.delete');
+
```

Comment:
Yep translation implementation looks good, but you also need to remove the other way of configuring the button text, this should be

```ts
const getButtonText = t('podDrawer.buttons.delete');
```

Reply:
🤔 so how can I change the `Delete Pod` text into `Restart pod`?